test/scripts/e2e_client_runner.py

```diff
@@ -353,6 +360,8 @@ def main():
     retcode = 0
     xrun(['goal', 'network', 'create', '-r', netdir, '-n', 'tbd', '-t', os.path.join(gopath, 'src/github.com/algorand/go-algorand/test/testdata/nettemplates/TwoNodes50EachFuture.json')], timeout=30)
     xrun(['goal', 'network', 'start', '-r', netdir], timeout=30)
+    if not args.keep_temps:
```

Comment:
don't need to specifically delete these because the entire tmpdir they live in is deleted by the above
``` Python
if not args.keep_temps:
   atexit.register(shutil.rmtree, tempdir, onerror=logger.error)
```